jenkinsci/jenkins
# Use a card layout instead of a table for the dashboard on mobile

**Before**
<img width="573" alt="image" src="https://user-images.githubusercontent.com/43062514/212175002-fcb70832-ee63-4e4b-9779-41238a2b196e.png">

**After**
<img width="543" alt="image" src="https://user-images.githubusercontent.com/43062514/219966834-0d51660e-f2e9-4be5-a91e-c5a25e7792ce.png">

Updates the dashboard to use a list of cards rather than a table when on a mobile device (or device with a low enough resolution). The layout obviously isn't as dense (or information rich) but it should be a lot more usable when running on smaller devices.

### Testing done

* Open the dashboard on a large browser, notice the table view looks the same as it always has. Resize the browser to be small, notice the table has been replaced with a list of cards
* Open a [folder](https://plugins.jenkins.io/cloudbees-folder/), notice the same behaviour as above

### Proposed changelog entries

- Use a card layout instead of a table for the dashboard on mobile.



### Proposed upgrade guidelines

N/A

## Proposed developer guidelines
* Plugins can now include a compactColumn for the dashboard in mobile view

### Submitter checklist

  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.

### Desired reviewers

@jenkinsci/sig-ux 



### Maintainer checklist

Before the changes are marked as `ready-for-merge`:



<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7581"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
